scripts/config/create-config.ts

```diff
@@ -0,0 +1,221 @@
+    .flatMap((beacon: any) =>
+      Object.entries(beacon.chains).map(([chainName, chain]: any) => {
+        const chainId = parseInt(operationsRepository.chains[chainName].id);
+        const dapiServerInteface = getDapiServerInterface();
+        const parameters = '0x';
+        const airnodeAddress = airnode;
+        const templateParameters = encode(beacon.template.decodedParameters);
+        const endpointId = ethers.utils.keccak256(
+          ethers.utils.defaultAbiCoder.encode(
+            ['string', 'string'],
+            [beacon.template.oisTitle, beacon.template.endpointName]
+          )
+        );
+        const templateId = ethers.utils.solidityKeccak256(['bytes32', 'bytes'], [endpointId, templateParameters]);
+        const threshold = ethers.BigNumber.from(100000000)
+          .mul(chain.updateConditionPercentage * 100)
+          .div(10000);
+        const beaconUpdateSubscriptionConditionParameters = ethers.utils.defaultAbiCoder.encode(
+          ['uint256'],
+          [threshold]
+        );
+        const encodedBeaconUpdateSubscriptionConditions = encode([
+          {
+            type: 'bytes32',
+            name: '_conditionFunctionId',
+            value: ethers.utils.defaultAbiCoder.encode(
+              ['bytes4'],
+              [dapiServerInteface.getSighash('conditionPspBeaconUpdate')]
+            ),
+          },
+          { type: 'bytes', name: '_conditionParameters', value: beaconUpdateSubscriptionConditionParameters },
+        ]);
+        const dapiServerAddress = operationsRepository.chains[chainName].contracts.DapiServer;
+        const sponsor = chain.sponsor;
+        const beaconUpdateSubscriptionId = ethers.utils.keccak256(
+          ethers.utils.defaultAbiCoder.encode(
+            ['uint256', 'address', 'bytes32', 'bytes', 'bytes', 'address', 'address', 'address', 'bytes4'],
+            [
+              chainId,
+              airnodeAddress,
+              templateId,
+              parameters,
+              encodedBeaconUpdateSubscriptionConditions,
+              airnodeAddress,
+              sponsor,
+              dapiServerAddress,
+              dapiServerInteface.getSighash('fulfillPspBeaconUpdate'),
+            ]
+          )
+        );
+        return {
+          [beaconUpdateSubscriptionId]: {
+            chainId: `${chainId}`,
+            parameters,
+            airnodeAddress,
+            templateId,
+            conditions: encodedBeaconUpdateSubscriptionConditions,
+            relayer: airnodeAddress,
+            sponsor,
+            requester: dapiServerAddress,
+            fulfillFunctionId: dapiServerInteface.getSighash('fulfillPspBeaconUpdate'),
+          },
+        };
+      })
+    )
+    .reduce((subscriptionsObject, subscription) => ({ ...subscriptionsObject, ...subscription }), {});
+
+  const airkeeperTriggers: Triggers = {
+    rrpBeaconServerKeeperJobs: [],
```

Comment:
Should we add a TODO here for these triggers?

Reply:
yeah good idea